CanvasPlusPlayground/Features/People/PeopleManager.swift

```diff
@@ -12,12 +12,19 @@ import SwiftData
 class PeopleManager {
     private let courseID: String?
 
-    var enrollments = [Enrollment]()
+    private var enrollments = [Enrollment]()
+
     var users: [User] {
-        Set(enrollments.compactMap(\.user))
-            .sorted {
-                ($0.name ?? "") < ($1.name ?? "")
-            }
+        Set(
+            enrollments
+                .compactMap {
+                    guard var user = $0.user else { return nil }
+                    user.role = $0.displayRole
```

Comment:
I feel like we could just reference `displayRole` directly where needed. Is there a reason for why we set `role` here?

Reply:
I think for now it's better to keep the `users` exposed to the view rather than `enrollments`, which have a lot more properties than necessary.

Reply:
Oh didn't realize that displayRole was for enrollments